pyclam/manifold.py

```diff
@@ -133,13 +135,15 @@ def argsamples(self) -> Vector:
             else:
                 n = int(np.sqrt(len(self)))
                 indices = list(np.random.choice(self.argpoints, n, replace=False))
+                indices = [int(i) for i in indices]
```

Comment:
can't we do something like `random.choice(dtype=int)`

Reply:
I think numpy just maps that to np.int from native ints